client/js/lounge.js

```diff
@@ -585,7 +585,20 @@ $(function() {
 			// https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/for...in#Iterating_over_own_properties_only
 			for (let key in params) {
 				if (params.hasOwnProperty(key)) {
-					const value = params[key];
+					let value = params[key];
+
+					if (key === "join") {
+						let channels = value.split(",");
+						channels = channels.map((c) => {
+							if (c.match(/^\w/)) {
+								return "#" + c;
+							}
+
+							return c;
+						});
+						value = channels.join(",");
```

Comment:
You can chain `value = split.map.join`.

Reply:
Good shout, changed

Reply:
Did you push the change? Doesn't appear to be changed.

Reply:
Would seem I didn't `git add` before amending the commit. Fixed now in 25dee77